README.md

```diff
@@ -1,3 +1,244 @@
-# Backend app
+# Backend app - Music Library
 
-This app was created with `@command-shift/create-backend-app`
+This app was created with `@command-shift/create-backend-app`. It is a music library API created with the CRUD REST API operations using Node.js, Express, Mocha-Chai, pgAdmin, and Postman.
+
+## Concepts of the app
+
+- Database migration and design
+- PostgreSQL
+- Creating and building databases in Docker container
+- Creating APIs with CRUD operations on databases
+- Integration testing with Mocha, Chai, and supertest
+- Using Postman to manage API requests
+- Nodemon to automatically restart app whenever changes are made to code
+- Using .dotenv to secure sensitive information
+
+## Getting started
+
+Fork and clone the repo, then run the following commands. Use pgAdmin and Postman to check whether CRUD operations and REST APIs are working
+
+```bash
+npm test # to test the code
```

Comment:
Before a user can run these commands they would have to run `npm install` to install the project's dependencies. They would also have to create a .env and .env.test file